### PR TITLE
Desktop design

### PR DESCRIPTION
**Pull request by kaizipaul**
In this PR:
-- Added elements to HTML file to accommodate desktop design
-- Added media query to CSS file to accommodate desktop and mobile design
-- Used flexbox and grid to make desktop design dynamic

**Thank you for reviewing my PR**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -188,6 +210,7 @@ <h5>Skills</h5>
     <!--forms section-->
 
     <section class="form-container">
+        <div class="form-flex">
       <h2 class="form-intro">
         I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
       </h2>
```
Comment (awais-amjed) [inline index.html:264]: Kindly align the left text with the Top of the Contact form as shown in Figma Design 😃

Your Design:
image

Figma Design:
image

**Review by awais-amjed — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️

Hi @kaizipaul , 

Good job so far! 👏👏
There are some issues that you still need to work on to go to the next project but you are almost there! 🥳🎈✨🙌

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] No Linter issues ✔️ 
- [x] Nice Design ✔️
- [x] Good code organization ✔️ 

gif

## Required Changes ♻️

- [x] Kindly note that for each project
  - _When the project is hovered on with the mouse, the description and dark opacity disappear and a See Project button appears at the bottom of the image and takes the whole image width._
    Your design:
image
    Figma Design:
image

- [x] Also note that The featured project is followed by 2 rows with 3 projects each (6 projects in total). So kindly add 3 more projects 🤗

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — CHANGES_REQUESTED** [review]
# **Status: Required Changes** :recycle: 

Hi @kaizipaul, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Top Highlight

-  No Linter errors. :heavy_check_mark:
-  Descriptive Pull Request. :heavy_check_mark:
 -  Nice Design. :heavy_check_mark:
 -  Good code organization, :heavy_check_mark: 
 - Meaningful commit messages. :heavy_check_mark: 




## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

- [x] As the previous reviewer pointed out. 
     -  When the project is hovered on with the mouse, the description and dark opacity disappear and a See Project button appears at the bottom of the image and takes the whole image width. I can see that this required changes has not been implemented.

### Figma Design

figma-design-desktop-ver


### Your Design

your-design-desktop-ver

- [x] Another change required by the previous reviewer is that the featured project is followed by 2 rows with 3 projects each (6 projects in total). Kindly add 3 more projects. This change required has not been implemented too. Kindly attend to this too. You are almost there trust me :mechanical_arm:

**Review by ShahierNashaat — APPROVED** [review]
Hi @kaizipaul, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Good overall design ✔️
- Your code is clean and well documented ✔️

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**